ReviewBot.py

```diff
@@ -179,7 +179,13 @@ def check_requests(self):
             if override is not None:
                 good = override
             else:
-                good = self.check_one_request(req)
+                try:
+                    good = self.check_one_request(req)
+                except:
```

Comment:
catching all exceptions is nasty actually. It would hide all problems. It's better to really only catch stuff from urllib.

Reply:
It accomplishes the goal of now blocking review of other requests. Exceptions should be alerted on and then resolved outside of that flow.